inductiva/tasks/task.py

```diff
@@ -14,6 +14,7 @@
 from inductiva.utils import files
 from inductiva.utils import data
 from inductiva.utils import output_contents
+from inductiva import types
```

Comment:
Check Hugo's emails about how to do import: avoid directly importing classes and functions.

Also, it's ok to just import inductiva and then have the full path so we can get the full package context in the code.

Reply:
Importing the full paths isn't going to be messy? For example doing inductiva.client.apis.tags.tasks_api.TasksApi

Reply:
Importing the full paths isn't going to be messy? For example doing `inductiva.client.apis.tags.tasks_api.TasksApi()` would be super long.

Reply:
Yes, I see your point! I will open an issue about it, since I think this PR is not the place to do that change.